rpc/src/transaction_status_service.rs

```diff
@@ -94,93 +94,94 @@ impl TransactionStatusService {
                     token_balances.post_token_balances,
                     transaction_indexes,
                 ) {
-                    if let Ok(committed_tx) = commit_result {
-                        let CommittedTransaction {
-                            execution_details:
-                                TransactionExecutionDetails {
-                                    status,
-                                    log_messages,
-                                    inner_instructions,
-                                    return_data,
-                                    executed_units,
-                                    ..
-                                },
-                            fee_details,
-                            rent_debits,
-                            ..
-                        } = committed_tx;
-                        let tx_account_locks = transaction.get_account_locks_unchecked();
-
-                        let fee = fee_details.total_fee();
-                        let inner_instructions = inner_instructions.map(|inner_instructions| {
-                            map_inner_instructions(inner_instructions).collect()
-                        });
-
-                        let pre_token_balances = Some(pre_token_balances);
-                        let post_token_balances = Some(post_token_balances);
-                        let rewards = Some(
-                            rent_debits
-                                .into_unordered_rewards_iter()
-                                .map(|(pubkey, reward_info)| Reward {
-                                    pubkey: pubkey.to_string(),
-                                    lamports: reward_info.lamports,
-                                    post_balance: reward_info.post_balance,
-                                    reward_type: Some(reward_info.reward_type),
-                                    commission: reward_info.commission,
-                                })
-                                .collect(),
+                    let Ok(committed_tx) = commit_result else {
```

Comment:
these few lines are the only real change. The rest of the diff is just changing the indent